src/account/AccountStorage.sol

```diff
@@ -63,21 +65,14 @@ struct AccountStorage {
     mapping(address => PluginData) pluginData;
     // Execution functions and their associated functions
     mapping(bytes4 => SelectorData) selectorData;
+    mapping(FunctionReference => ValidationData) validationData;
```

Comment:
This is the `defaultValidation` pool, right? Should we rename it?

Reply:
Let's also start using named parameters for mappings to make things clearer. For example:
```solidity
    mapping(FunctionReference validationFunction => ValidationData) ...
```

Reply:
It holds some other data per validation function, too. Specifically, whether or not the validation function is allowed to do signature validation (1271's `isValidSignature`) and the set of pre-validation hooks that apply to the validation function.

Also I added the named mapping parameter `validationFunction`.